.profile.d/ruby-sidekiq-metrics.sh

```diff
@@ -21,10 +21,7 @@ setup_metrics() {
   fi
 
   echo "sidekiq-metrics start..."
-  (while true; do
-     "$SIDEKIQ_METRICS"
-     sleep "$SIDEKIQ_METRICS_INTERVAL"
-   done) &
+  "$SIDEKIQ_METRICS" "$SIDEKIQ_METRICS_INTERVAL" &
```

Comment:
It does not work well.

It seems the output does not print to Papertrail until Dyno shutdown.
But I set `$stdout.sync = true` in sidekiq-metrics. Why?